cli/Valet/Site.php

```diff
@@ -710,9 +710,10 @@ function unsecureAll()
      *
      * @param  string  $url The domain name to serve
      * @param  string  $host The URL to proxy to, eg: http://127.0.0.1:8080
+     * @param  bool    $unsecure
      * @return string
      */
-    function proxyCreate($url, $host)
+    function proxyCreate($url, $host, $unsecure = false)
```

Comment:
Ok, I'll update the parameter and variable.

Just a question, `insecure` may seem pejorative. Maybe it should be about protocol : http v https ?

Reply:
If I'm honest, I don't like this change to `insecure` at all. I'd rather we rollback this latest.

Valet's convention for other sites (not proxy), is `valet.conf` and `secure.valet.conf`. When it handles securing something, that's when it uses wording/terms related to `secure`. Otherwise it assumes not-secure.

I did break that convention slightly when I added the proxy option, *specifically* because of all the complexities that this PR highlights. But, here we are. 
If we're going ahead with this PR, then I'd suggest rolling back all these `insecure` rewording changes.

Reply:
@drbyte can you explain that a bit?

I think there are *two* changes I suggested here:

1. prefixing `valet.conf` with `insecure`, which, if it breaks convention, is totally fine to roll back!
2. Changing `unsecure` to `insecure` *if* this is referring to the word being used as a *description* (adjective) instead of a *command* (verb).

Sounds like we should rollback the prefixing of the file.

Can anyone explain to me: is `unsecure` an action we're taking? "Unsecure this site!" ? Or is it a description of *how* we're doing something, adverbially? "Do this action, but in an insecure way". That is what defines which word it should be for me.

Reply:
In a general sense, if we think about the concept "outside of this proxy-related PR", Valet uses the words "secure" and "unsecure" as actions, to add/remove SSL on a site.

Internally some class methods therefore are related to actions taken to fulfill those requests.
In practice, the action of "creating a proxy-site using a non-secured proxy stub" must for safety/cleanup reasons first ensure that the site is `unsecure`'d in case that site had been "secured" in the past. So `unsecure` as a verb for the method call makes sense.
Thus I don't think we need `putInsecurely()` etc. IMO `put()` is fine, and `secure()`/`unsecure()` are fine.

Where `$unsecure` is used as a method *parameter* (such as in `proxyCreate()`, maybe we can rename it to `$secure` and reverse the logic accordingly, to minimize confusion.

As for stub filename, I don't think we need the `insecure.` prefix on the filename.

Reply:
- I'm totally fine without the prefix on the filename
- I definitely think `secure` and `unsecure` are good method names
- totally get the reason for wanting to call `unsecure` before creating a proxy-site using a non-secured proxy stub, and therefore totally understand the not needing to rename to `putInsecurely`
- I'm OK with renaming it to `$secure` in the method list and then inverting it, think that's a great idea

That only leaves the actual command line flag, which I still think makes sense to be `--insecure`, but what do you think?

Reply:
How about `--secure` for CLI too? ie: defaults to non-secure, unless --secure is specified?

Reply:
hm, that's a good point! It's better language *and* behaves more than the rest of the site.

I think the only downside would be, if I have my app secured, and then I run `valet share`, I don't think I would expect it to suddenly un-secure it.. right? that's a problem either way though. I AM LOST AND CONFUSED

Reply:
I thought we were only talking about `valet proxy --secure`, not `valet share`

Reply:
@drbyte Oh crap. You're absolutely right. YES. I like that. Sorry!